src/nv_ingest/schemas/ingest_job_schema.py

```diff
@@ -43,6 +43,7 @@ class TaskTypeEnum(str, Enum):
     filter = "filter"
     split = "split"
     store = "store"
```

Comment:
in an effort to make the minimal number of changes, I opted to leave the store_images behavior under the `store` tag. Although now that multiple store actions exist, that is probably something we might want to update to more accurate depict the action completed by a task.

Reply:
Sounds like a good idea to rename this.

Reply:
maybe we can do that in a subsequent PR so that this does not get more bloated?

Reply:
Yes.